packages/vite/src/client/client.ts

```diff
@@ -558,9 +558,12 @@ export function createHotContext(ownerPath: string): ViteHotContext {
     decline() {},
 
     // tell the server to re-perform hmr propagation from this module as root
-    invalidate() {
-      notifyListeners('vite:invalidate', { path: ownerPath })
-      this.send('vite:invalidate', { path: ownerPath })
+    invalidate(message) {
+      notifyListeners('vite:invalidate', { path: ownerPath, message })
+      this.send('vite:invalidate', { path: ownerPath, message })
+      console.log(
+        `[vite] invalidate ${ownerPath}${message ? `: ${message}` : ''}`
+      )
```

Comment:
We moved the HMR updates messages to the debug channel in the browser so they are only visible if people opt-in. Since we have the message in the terminal, I think it is ok to do the same here, no? 
```suggestion
      console.debug(
        `[vite] invalidate ${ownerPath}${message ? `: ${message}` : ''}`
      )
```

Reply:
Yeah after reflexion it's best to focus having nice messages on the terminal and debug only in the browser, so that user knows where to look for logs by default